components/aws/sagemaker/tests/unit_tests/tests/Endpoint/test_Endpoint_component.py

```diff
@@ -0,0 +1,199 @@
+from commonv2.sagemaker_component import SageMakerJobStatus
+
+from Endpoint.src.Endpoint_spec import SageMakerEndpointSpec
+from Endpoint.src.Endpoint_component import SageMakerEndpointComponent
+import unittest
+
+from unittest.mock import patch, MagicMock
+
+
+class EndpointComponentTestCase(unittest.TestCase):
+    REQUIRED_ARGS = [
+        "--region",
+        "us-west-1",
+        "--endpoint_config_name",
+        "test",
+        "--endpoint_name",
+        "test1",
```

Comment:
nit: Can we have a more descriptive name for the test than test1?

Reply:
Done